src/main/java/Angela/Angela.java

```diff
@@ -0,0 +1,190 @@
+package Angela;
+
+import Angela.util.Command;
+
+import Angela.storage.Database;
+import Angela.storage.TaskList;
+
+
+import java.util.Scanner;
+
+/**
+ * Main class of Angela
+ */
+public class Angela {
+
+    // private elements
+    private Database database;
+    private TaskList listData;
+
+    /**
+     * Constructs an instance of the Angela class with the specified file path.
+     * This constructor initializes the `listData` and `database` fields after a delay of 10 seconds.
+     * The `listData` is initialized as a new `TaskList` object, and the `database` is initialized
+     * as a new `Database` object using the provided file path and the `listData`.
+     *
+     * @param filePath The path to the file that will be used by the `Database` for storage or retrieval.
+     *                 This path is passed to the `Database` constructor.
+     * @see TaskList
+     * @see Database
+     */
+    public Angela(String filePath) {
+        setTimeout(() -> {
+            this.listData = new TaskList();
+            this.database = new Database(filePath, listData);
+        },10000);
```

Comment:
```suggestion
        }, 10000);
```
Commas should be followed by a white space.